daemon/logger/syslog/syslog.go

```diff
@@ -0,0 +1,55 @@
+package syslog
+
+import (
+	"log/syslog"
+	"os"
+	"path"
+
+	"github.com/docker/docker/daemon/logger"
+)
+
+type Syslog struct {
+	writer *syslog.Writer
+	tag    string
+}
+
+func New(tag string) (logger.Logger, error) {
+	return &Syslog{
+		writer: nil,
+		tag:    tag,
+	}, nil
+}
+
+func (s *Syslog) Log(msg *logger.Message) error {
+	if s.writer == nil {
+		log, err := syslog.New(syslog.LOG_USER, path.Base(os.Args[0]))
+		if err != nil {
+			return err
+		}
+		s.writer = log
+	}
+	if msg.Source == "stderr" {
+		if err := s.writer.Err(s.tag + " " + string(msg.Line)); err != nil {
```

Comment:
can we change to 

    s.tag + ": " +

This way the message looks more like `docker[23348]: d83f42d92a0e: hi`